EIPS/eip-6956.md

```diff
@@ -0,0 +1,515 @@
+---
+eip: 6956
+title: Asset-bound Non-Fungible Tokens
+description: Asset-bound NFTs anchor a token 1-1 to an asset and transfers are authorized through oracle-attestation of control over the asset
+author: Thomas Bergmueller (@tbergmueller) <thomas.bergmueller@authenticvision.com>, Lukas Meyer (@ibex-technology) <lukas@ibex.host>
+discussions-to: https://ethereum-magicians.org/t/erc-6956-asset-bound-non-fungible-tokens/14056
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-04-29
+requires: 165, 721
+---
+
+## Abstract
+
+This standard allows to integrate physical and digital ASSETS without signing capabilities into dApps/web3 by extending [ERC-721](eip-721.md).
+
+An `ASSET`, e.g. a physical object, is equipped with an `ANCHOR`. The `ANCHOR-TECHNOLOGY` must be chosen s.t. an ANCHOR allows to uniquely identify the ASSET. The ANCHOR-TECHNOLOGY must further allow to establish a `PROOF-OF-CONTROL` over the ASSET through an `ORACLE`. For physical ASSETS, PROOF-OF-CONTROL corresponds for example to proof of physical presence.
+
+The ANCHOR is mapped 1:1 to a tokenId on-chain, hence represents each individual ASSET 1:1.
+Mapping in a secure, inseperable manner requires the ORACLE to issue an off-chain signed `ATTESTATION`, which is on-chain-verifyable. Through the ATTESTATION, the ORACLE testifies that a particular ASSET associated with an ANCHOR has been `CONTROLLED` when defining a `to`-address, e.g. through a user-device.
+
+This standard to proposes to use `ATTESTATIONS` as authorization for the following [ERC-721](eip-721.md) mechanisms: `transfer`, `burn` and `approve`. The proposed `transferAnchor(attestation)`, `burnAnchor(attestation)` and `approveAnchor(attestation)` are permissionless, i.e. neither the sender/owner (`from`) nor the receiver (`to`) need to sign. Authorization is solely provided through the ORACLE's ATTESTATION.
+
+We also outline for optional use
+
+- a `FLOATING`-concept (temporarily or permanently enabling "traditional" [ERC-721](eip-721.md) transfers without ATTESTATION)
+- `ATTESTATION-LIMITS`, which are recommended to implement for security reasons, when gas is paid through a central account (see Figure 1)
+
+Figure 1 below shows a the data flow of an asset-bound NFT transfer through a simplified example system employing the proposed standard. The system is utilizing a smartphone as user-device to interact with a physical ASSET.
+
+![Figure 1: Sample system](../assets/eip-6956/img/erc6956_concept.svg)
+
```

Comment:
Your abstract is a tad on the long side.

Reply:
As I mentioned in Office Hours #17 its a bit hard to shorten since the EIP itself is long and slightly complex. Managed to reduce from 250 words to 199 in https://github.com/ethereum/EIPs/commit/9d188578511cb53c00ebe0910043ab50fb40a248 .